dask_planner/src/expression.rs

```diff
@@ -625,7 +625,7 @@ impl PyExpr {
     pub fn bool_value(&mut self) -> PyResult<bool> {
         match &self.expr {
             Expr::Literal(scalar_value) => match scalar_value {
-                ScalarValue::Boolean(iv) => Ok(iv.unwrap()),
+                ScalarValue::Boolean(Some(iv)) => Ok(*iv),
```

Comment:
@charlesbluca your question is related to the change here - originally, "IS UNKNOWN" would panic here when the value is null, then throws the type error on the next line.

Not sure if we can come up with a case where it wouldn't be `Some(true)`, `Some(false)`, or `None`